lib/emscripten/src/syscalls/unix.rs

```diff
@@ -418,30 +420,21 @@ pub fn ___syscall330(ctx: &mut Ctx, _which: c_int, mut varargs: VarArgs) -> pid_
 /// ioctl
 pub fn ___syscall54(ctx: &mut Ctx, _which: c_int, mut varargs: VarArgs) -> c_int {
     debug!("emscripten::___syscall54 (ioctl) {}", _which);
+
     let fd: i32 = varargs.get(ctx);
     let request: u32 = varargs.get(ctx);
     debug!("=> fd: {}, op: {}", fd, request);
+
     // Got the equivalents here: https://code.woboq.org/linux/linux/include/uapi/asm-generic/ioctls.h.html
-    // let argp: u32 = varargs.get(ctx);
-    // let argp_ptr = emscripten_memory_pointer!(ctx.memory(0), argp) as *mut c_void;
+    let argp: u32 = varargs.get(ctx);
+    let argp_ptr = emscripten_memory_pointer!(ctx.memory(0), argp) as *mut c_void;
     // let ret = unsafe { ioctl(fd, request as _, argp_ptr) };
     // debug!("=> {}", ret);
     // ret
+
     match request as _ {
-        21537 => {
-            // FIONBIO
-            let argp: u32 = varargs.get(ctx);
-            let argp_ptr = emscripten_memory_pointer!(ctx.memory(0), argp) as *mut c_void;
-            let ret = unsafe { ioctl(fd, FIONBIO, argp_ptr) };
-            debug!("ret(FIONBIO): {}", ret);
-            ret
-            // 0
-        }
-        21523 => {
-            // TIOCGWINSZ
-            let argp: u32 = varargs.get(ctx);
-            let argp_ptr = emscripten_memory_pointer!(ctx.memory(0), argp) as *mut c_void;
-            let ret = unsafe { ioctl(fd, TIOCGWINSZ, argp_ptr) };
+        TIOCGWINSZ => {
```

Comment:
The value of `TIOCGWINSZ` is different per OS in libc:
https://github.com/rust-lang/libc/search?q=TIOCGWINSZ&unscoped_q=TIOCGWINSZ

Since the WebAssembly is compiled with a fixed value, the value here should be fixed too.

```suggestion
        21523 => {
```

Reply:
Good catch, really, didn't thought about that point and instead focused on readibility. Ok, I'll remove the value fetch from `libc` and set it as a constact at file top.